cli/src/commands/dbs/delete.ts

```diff
@@ -25,12 +25,15 @@ export default class DatabasesDelete extends BaseCommand {
 
     const xata = await this.getXataClient();
 
-    const { confirm } = await prompts({
-      type: 'confirm',
-      name: 'confirm',
-      message: `Are you sure you want to delete database ${workspace}/${database}?`,
-      initial: true
-    });
+    const { confirm } = await this.prompt(
+      {
+        type: 'confirm',
+        name: 'confirm',
+        message: `Are you sure you want to delete database ${workspace}/${database}?`,
```

Comment:
To be consistent with earlier messages:


```suggestion
        message: `Are you sure you want to delete database ${database} in the ${workspace} workspace?`,
```